src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -412,7 +412,11 @@ private function configureFileValidator(array $config, ContainerBuilder $contain
         $definition = $container->getDefinition('sulu_media.file_validator');
         $definition->addMethodCall('setMaxFileSize', [$config['upload']['max_filesize'] . 'MB']);
 
-        $blockedFileTypes = $config['format_manager']['blocked_file_types'];
+        $blockedFileTypes = $config['upload']['blocked_file_types'];
+        if (0 === \count($blockedFileTypes)) {
+            $blockedFileTypes = $config['format_manager']['blocked_file_types'];
```

Comment:
there is also a parameter set with `sulu_media.media.blocked_file_types` which should also contain the new value. Not sure why not this parameter was use in the service definition 🙈.